### ISSUE TITLE
Basic Simulation Functionality

### ISSUE THREAD

**Issue by jebidiah252** [jak103/powerscore#6]
We need to create the basic simulation functionality to show that a game can be simulated.

Once we have both frontends started (scorekeeper can hit some buttons and an event will fire), we can start working on the integration needed to see game updates in real time.

An idea could be to setup a basic web socket connection between the two different frontends, with the scorekeeper having a button that would generate a web socket message and the scoreboard frontend listening for that message. That message could contain some basic info that says to update the home team's score by one, or to increase the period by one.

**Comment by jebidiah252** [comment]
Changing the size of this ticket from Medium to Large. I'm thinking this is going to be lots of integration work between the frontends.

**Comment by Potato-Man114** [comment]
@jebidiah252 Is this to simulate actions taken by scorekeepers and its effect, or is this to simulate data coming from the ISC device?

**Comment by jebidiah252** [comment]
> [@jebidiah252](https://github.com/jebidiah252) Is this to simulate actions taken by scorekeepers and its effect, or is this to simulate data coming from the ISC device?

This issue is meant for the former. We'd want to simulate what happens when a scorekeeper uses their UI to update the scoreboard.

**Comment by Potato-Man114** [comment]
If that's the case, then this large ticket should be broken up into smaller ones.
- Connect Scorekeeper and Live Game scoreboard interfaces.
- Create a simulation of the scorekeeper updating the scoreboard via the web interface (Cypress?)
- Create a simulation to demonstrate reading data from the ISC 9000 (probably a lower priority.)

What are your thoughts on that, @jebidiah252 ?

**Comment by jebidiah252** [comment]
> If that's the case, then this large ticket should be broken up into smaller ones.
> 
> * Connect Scorekeeper and Live Game scoreboard interfaces.
> * Create a simulation of the scorekeeper updating the scoreboard via the web interface (Cypress?)
> * Create a simulation to demonstrate reading data from the ISC 9000 (probably a lower priority.)
> 
> What are your thoughts on that, @jebidiah252 ?

Yeah, that's a great idea. I set the size on this issue to XL as a way to mark that it's going to be a lot of work and that sub tasks would probably be needed. That's a fault on my part though and not communicating that.

**Comment by Potato-Man114** [comment]
> Yeah, that's a great idea. I set the size on this issue to XL as a way to mark that it's going to be a lot of work and that sub tasks would probably be needed. That's a fault on my part though and not communicating that.

Awesome, I'll create those tickets, then.
